src/grpc.js

```diff
@@ -0,0 +1,72 @@
+const { ChunkParser, ChunkType } = require("grpc-web-client/dist/ChunkParser") 
+
+export function grpcJSONRequest(host, packageName, serviceName, methodName, requestHeaders, requestObject) {
+  const service = [ packageName, serviceName ].filter(Boolean).join(".")
+  return window.fetch(`${host}/${service}/${methodName}`, {
+    "method": "POST",
+    "headers": Object.assign(
+      {},
+      {
+        "content-type": "application/grpc-web+json",
+        "x-grpc-web": "1"
+      },
+      requestHeaders
+    ),
+    "body": frameRequest(Buffer.from(JSON.stringify(requestObject)))
+  })
+    .then(response => response.arrayBuffer())
+    .then(buffer => {
+      return grpcJSONResponseToString(buffer)
+    })
+    .catch(console.error)
+}
+
+export function rpcImpl(host, packageName, serviceName, methodName, requestHeaders) {
+  return (method, requestObject, callback) => {
+    const service = [ packageName, serviceName ].filter(Boolean).join(".")
+    window.fetch(`${host}/${service}/methodName`, {
```

Comment:
I'm getting error here.
I think this should be: `window.fetch(`${host}/${service}/${methodName}`, {`